docs: updating readme and version for rebranding (#366)
chanzuckerberg/single-cell-explorer: README.md

# CZ CELLxGENE Explorer
_an interactive explorer for single-cell transcriptomics data_

This repo contains a fork of CZ CELLxGENE Annotate called CZ CELLxGENE Explorer which is deployed as part of CZ CELLxGENE Discover to enable exploration of datasets published on that platform.

[![Push Tests](https://github.com/chanzuckerberg/single-cell-explorer/workflows/Push%20Tests/badge.svg)](https://github.com/chanzuckerberg/single-cell-explorer/actions?query=workflow%3A%22Push+Tests%22)
![Code Coverage](https://codecov.io/gh/chanzuckerberg/single-cell-explorer/branch/main/graph/badge.svg)

> CZ CELLxGENE (pronounced "cell-by-gene") is an interactive data explorer for single-cell transcriptomics datasets, such as those coming from the [Human Cell Atlas](https://humancellatlas.org). Leveraging modern web development techniques to enable fast visualizations of at least 1 million cells, we hope to enable biologists and computational researchers to explore their data.

Reviewer: ```suggestion
CZ CELLxGENE Explorer (pronounced "cell-by-gene") is an interactive data explorer for single-cell transcriptomics datasets, such as those coming from the [Human Cell Atlas](https://humancellatlas.org). Leveraging modern web development techniques to enable fast visualizations of at least 1 million cells, we hope to enable biologists and computational researchers to explore their data.
```

> Whether you need to visualize one thousand cells or one million, CZ CELLxGENE helps you gain insight into your single-cell data.

Reviewer: ```suggestion
Whether you need to visualize one thousand cells or one million, CELLxGENE Explorer helps you gain insight into your single-cell data.
```

Per comments elsewhere, we're trying to establish CELLxGENE as the brand family, so are eliminating uses of that name to refer to individual applications. Here the correct referent is "Explorer"

## Getting started

### The comprehensive guide to CZ CELLxGENE

> [The CZ CELLxGENE documentation is your one-stop-shop for information about CELLxGENE](cellxgene.cziscience.com/docs)!

Reviewer 2: Let's swap to a relative `/docs`

> [The CZ CELLxGENE documentation is your one-stop-shop for information about CELLxGENE](https://github.com/chanzuckerberg/cellxgene-documentation/blob/main/README.md)!

Reviewer: Whoops, this link is outdated. Sneaking in this change:

```suggestion
[The CZ CELLxGENE documentation is your one-stop-shop for information about CELLxGENE](cellxgene.cziscience.com/docs)!
```

### Supported browsers

> CZ CELLxGENE currently supports the following browsers:

Reviewer: ```suggestion
CZ CELLxGENE Explorer currently supports the following browsers:
```
```suggestion
CZ CELLxGENE currently supports the following browsers:
```

For any errors, [report bugs on Github](https://github.com/chanzuckerberg/single-cell-explorer/issues/new/choose).

> # Developing with CZ CELLxGENE

Reviewer: ```suggestion
# Developing with CZ CELLxGENE Explorer
```

### Contributing

> Please see our [contributing guide](https://github.com/chanzuckerberg/cellxgene-documentation/blob/main/contribute.md) and don't hesitate to open an issue or send a pull request to improve CZ CELLxGENE. Please see the [dev_docs](https://github.com/chanzuckerberg/single-cell-explorer/blob/main/dev_docs) for pull request suggestions, unit test details, local documentation preview, and other development specifics. 

Reviewer: ```suggestion
Please see our [contributing guide](https://github.com/chanzuckerberg/cellxgene-documentation/blob/main/contribute.md) and don't hesitate to open an issue or send a pull request to improve CZ CELLxGENE Explorer. Please see the [dev_docs](https://github.com/chanzuckerberg/single-cell-explorer/blob/main/dev_docs) for pull request suggestions, unit test details, local documentation preview, and other development specifics. 
```

this project. All code is freely available for reuse under the [MIT license](https://opensource.org/licenses/MIT).

> Before extending CZ CELLxGENE, we encourage you to reach out to us with ideas or questions. It might be possible that an 

Reviewer: ```suggestion
Before extending CZ CELLxGENE Explorer, we encourage you to reach out to us with ideas or questions. It might be possible that an 
```
